tools/common/StaticRegistrar.cs

```diff
@@ -3309,6 +3309,12 @@ void Specialize (AutoIndentStringBuilder sb, out string initialization_method)
 
 		bool HasIntPtrBoolCtor (TypeDefinition type, List<Exception> exceptions)
 		{
+			return HasIntPtrBoolCtor (type, exceptions, out var _);
+		}
+
+		bool HasIntPtrBoolCtor (TypeDefinition type, List<Exception> exceptions, [NotNullWhen (true)] out MethodDefinition? ctor)
```

Comment:
minor: this flavor should probably be name `TryGetIntPtrBoolCtor`

Reply:
https://github.com/xamarin/xamarin-macios/pull/18730